account_financial_report_webkit/models/account_static_balance.py

```diff
@@ -36,13 +39,16 @@ def get_entries_to_calculate(self, periods=None):
         """
         query = """
           WITH balances AS (
-            SELECT ap.id as period, aa.id as account, asb.id as balance
+            SELECT ap.id as period, aa.id as account, asb.id as balance,
```

Comment:
You have a shadow data model on a production like database somewhere, right? Did you check the performance of this query with the added cross join on that data?

Reply:
Last time i checked, it took 1 hour and +- 45 minutes on UAT09. I will run it once again to validate the timing. But since we are also introducing the migration to big-integers what will take some time, Pieter suggested to handle the static balance calculation in jobs so we dont have a deploy window of > 4 hours. What do you think about it?

Reply:
With the new amount of records, the old query is extremely slow, and the extended query even slower. I am going to rewrite this function fully.

Reply:
Thanks for checking. It's what I was afraid of.